README.md

```diff
@@ -1,2 +1,24 @@
-# rails-blog
-This is a simple blog built with the power of Ruby on Rails
+# README
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
+
+Things you may want to cover:
+
+* Ruby version
+
+* System dependencies
+
+* Configuration
+
+* Database creation
+
+* Database initialization
+
+* How to run the test suite
+
+* Services (job queues, cache servers, search engines, etc.)
+
+* Deployment instructions
```

Comment:
- [ ] Kindly update your readme file to reflect the changes for this milestone, checkout this [template](https://github.com/microverseinc/readme-template) from Microverse and kindly adapt it to your project.